src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileMetadataService.scala

```diff
@@ -127,7 +127,7 @@ class FileMetadataService(
   }
 
   def getFileMetadata(consignmentId: UUID, selectedFileIds: Option[Set[UUID]] = None): Future[Map[UUID, FileMetadataValues]] =
```

Comment:
I think you need to make `consignmentId` argument an `Option` too and pass `None` from the `getPaginatedFiles()` method. https://github.com/nationalarchives/tdr-consignment-api/blob/master/src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala#L165C43-L165C58

`fileMetadata <- fileMetadataService.getFileMetadata(consignmentId, fileIds)`